### ISSUE TITLE
How to change the thickness of the Icon

### ISSUE THREAD

**Issue by Qyokizzzz** [akveo/react-native-ui-kitten#1735]
import styled from 'styled-components/native';
import { Icon } from '@ui-kitten/components';

export const StyledIcon = styled(Icon)`
  height: 24px;
  width: 24px;
  font-weight: 100;
`;

I can only change the size of Icon, not its thickness.

**Comment by greenfrvr** [comment]
Could you please provide more details about icons pack you're using? If we're talking about Eva Icons Pack it's not based on fonts, so there is no way to change its thickness with font properties.

**Comment by Qyokizzzz** [comment]
Yes, I am using Eva. May I know which icon pack supports adjusting thickness?

**Comment by Qyokizzzz** [comment]
Is there any other way to change the thickness of the eva icon?
